playground/src/server.ts

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import dotenv from 'dotenv-flow';
+dotenv.config();
+// eslint-disable-next-line import/first
+import echoHandler from '../api/echo';
+
+const app = express();
+const port = process.env.PLAYGROUND_SERVER_PORT || 3003;
+
+// @ts-ignore
+app.get('/api/echo', echoHandler);
+// @ts-ignore
+app.post('/api/echo', echoHandler);
+
+app.listen(port, () => {
+  console.log(`Playground server listening on port ${port}`);
+});
+
+module.exports = {};
```

Comment:
Is this needed? 🤔

Reply:
Sorry, this is a leftover.

Updated in this commit: a8f87d4a9dd0034e404e958535494d8c564cb997